frontend/src/app/app.component.spec.ts

```diff
@@ -18,10 +21,10 @@ describe('AppComponent', () => {
     expect(app.name).toEqual('frontend');
   });
 
-  it('should render title', () => {
-    const fixture = TestBed.createComponent(AppComponent);
-    fixture.detectChanges();
-    const compiled = fixture.nativeElement as HTMLElement;
-    expect(compiled.querySelector('.content span')?.textContent).toContain('frontend app is running!');
-  });
+  // it('should render title', () => {
```

Comment:
Would like to confirm if this should be left in here @itsrx @GenFusion122

Reply:
Oops, I will delete it. The app doesn't say this anymore so this test is redundant